test/parser/parse.test.ts

```diff
@@ -1855,3 +1857,40 @@ See more help with --help`)
     })
   })
 })
+
+describe('stdin', () => {
```

Comment:
Maybe you should add a test to verify that myFlag has the value of stdin? 


````js
...
const { flags } = this.parse(cmd)

assert(flags.myFlag).equals(stdinValue)
...
````

Reply:
I added an extra line in this test. Does that help?

`expect(out.flags.myflag).to.equals(stdinValue)`